src/particle/particle/kinematics.py

```diff
@@ -8,8 +8,9 @@
 """
 
 
-from hepunits.constants import hbar
-from hepunits.units import MeV, ns
+# Bug in HEPUnits exporting
+from hepunits.constants import hbar  # type: ignore[attr-defined]
+from hepunits.units import MeV, ns  # type: ignore[attr-defined]
```

Comment:
These type ignores would go away with a new HEPUnits

Reply:
In that case I can go ahead and immediately tag and release `hepunits` ... Off I go and will cc you ...

Reply:
Okay, I'll rebuild and remove the type ignores as soon as that goes out.

Reply:
`hepunits` 2. 3.0 is out - https://pypi.org/project/hepunits/2.2.0/